catalog/core/catalog-core-commands/src/test/groovy/org/codice/ddf/commands/catalog/ExportCommandSpec.groovy

```diff
@@ -84,6 +80,14 @@ class ExportCommandSpec extends spock.lang.Specification {
 
     def "Test export no items"() {
         setup:
+        catalogFramework.query(_ as QueryRequest) >> { QueryRequest req ->
+            System.err.println("catalogFramework.query() called!")
```

Comment:
I know we try to avoid `System.out`, but not sure about `System.err`.  Is this necessary?